packages/integrations/image/src/index.ts

```diff
@@ -27,11 +28,13 @@ export interface IntegrationOptions {
 	 * Entry point for the @type {HostedImageService} or @type {LocalImageService} to be used.
 	 */
 	serviceEntryPoint?: string;
+	logLevel?: LoggerLevel;
 }
 
 export default function integration(options: IntegrationOptions = {}): AstroIntegration {
 	const resolvedOptions = {
 		serviceEntryPoint: '@astrojs/image/sharp',
+		logLevel: 'info' as LoggerLevel,
```

Comment:
Thanks for this @tony-sull! Nice QOL improvement.

I like allowing this to be configured at the integration level to allow debugging _only_ images but do you think this should use a default set by the existing logging options?

Running `DEBUG=* astro build` or `DEBUG=vite:* astro build` is something we support via Vite, so maybe we should be tapping into that same system?

Reply:
Yeah I had considered that but ended up just mimicking core's logger for now.  Ideally we could expose Astro's internal logger to all integrations, there's a bit of work to really get that logger ready for outside use first though